src/firebaseAuthService.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+  "use strict";
+
+  function FirebaseAuthService($firebaseAuth, firebaseRef) {
+    return $firebaseAuth(firebaseRef);
+  }
+  FirebaseAuthService.$inject = ['$firebaseAuth', 'firebaseRef'];
```

Comment:
We don't use `$inject` anywhere else. Note that I don't like this approach or even find it more readable, but it's inconsistent, and if we're going to change the strategy, we should do that in a sep changelist.

The current format is: `angular.module('firebase').factory('$firebaseArray', ["$log", "$firebaseUtils", "$q",`